charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml

```diff
@@ -27,6 +27,10 @@ data:
     ChartName: cosi
     ChartVersion: 0.0.1-alpha.5
     RepositoryURL: '{{ if .Values.helmRepository.enabled }}oci://helm-repository.{{ .Release.Namespace }}.svc/charts{{ else }}https://mesosphere.github.io/charts/stable/{{ end }}'
+  distribution-registry-mirror: |
+    ChartName: docker-registry
```

Comment:
nit: original chart name could be `distribution-registry` :)

Reply:
Yeah :( , this name comes from the upstream chart https://github.com/twuni/docker-registry.helm/tree/main
It would have been too intrusive to change it and still be able to sync upstream changes.